### DISCUSSION TITLE
无法发表评论，发表评论弹出failed to fetch

### DISCUSSION THREAD

**Discussion by a1401358759** [walinejs/waline#702] [Q&A] [answered]
控制台报错：

Access to fetch at 'https://waline-5gsb5pbr4d74c22a-1256044091.ap-guangzhou.app.tcloudbase.com/waline/comment?lang=zh-CN' from origin 'https://yangsihan.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

是因为我的Nginx配置问题？

**Comment by a1401358759** [comment, marked as answer]
知道原因了 云开发CloudBase部署需要在腾讯云后台设置安全域名

**Reply by ShueyYuen** [reply]
鄙人部署在Vercel上遇到同样的问题，但是只在阅读量和评论数两个接口上出现问题，可以正常的发送评论等功能。

**Reply by heren1229** [reply]
> 鄙人部署在Vercel上遇到同样的问题，但是只在阅读量和评论数两个接口上出现问题，可以正常的发送评论等功能。

请问怎么解决呢

**Reply by ShueyYuen** [reply]
鄙人部署在vercel上，问题出现在从v1升级到v2，v1中的配置
```javascript
Waline.init({
    el: '#waline',
    serverURL: 'https://comment.example.com/',
})
```
升级到v2需要把serverURL后面的`/`去掉：
```javascript
Waline.init({
    el: '#waline',
    serverURL: 'https://comment.example.com',
})
```

**Reply by heren1229** [reply]
> 鄙人部署在vercel上，问题出现在从v1升级到v2，v1中的配置
> 
> ```js
> Waline.init({
>     el: '#waline',
>     serverURL: 'https://comment.example.com/',
> })
> ```
> 
> 升级到v2需要把serverURL后面的`/`去掉：
> 
> ```js
> Waline.init({
>     el: '#waline',
>     serverURL: 'https://comment.example.com',
> })
> ```

这个应该在哪改啊QAQ

**Reply by ShueyYuen** [reply]
参考官方文档的[waline客户端引用](https://waline.js.org/guide/client/import.html#%E4%BD%BF%E7%94%A8)

**Reply by heren1229** [reply]
> 参考官方文档的[waline客户端引用](https://waline.js.org/guide/client/import.html#%E4%BD%BF%E7%94%A8)

解决了,谢谢您(•̀⌄•́)

**Reply by Mister-Hope** [reply]
这个问题已经在 v2.5.2 修复，错误的原因是评论数和访问数接口没有校验 serverURL 格式

**Comment by who-win** [comment]
,,

**Comment by who-win** [comment]
zzzzzzzzzzzzzz
